bindata/network/openshift-sdn/alert-rules.yaml

```diff
@@ -39,3 +39,35 @@ spec:
       for: 1h
       labels:
         severity: warning
+    - alert: IPTableSyncSDNPod
+      annotations:
+        message: SDN pod {{"{{"}} $labls.pod {{"}}"}} on node {{"{{"}} labels.node {{"}}"}} takes too long to sync iptables rules.
+      expr: |
+        histogram_quantile(.95, kubeproxy_sync_proxy_rules_duration_seconds_bucket) * on(pod) group_right kube_pod_info{namespace="openshift-sdn",  pod=~"sdn-[^-]*"} > 15
+      labels:
+        severity: warning
+    - alert: IPTableSyncCluster
+      annotations:
+        message: The average time for SDN pods to sync iptables is too high.
+      expr: |
+        histogram_quantile(0.95, sum(rate(kubeproxy_sync_proxy_rules_duration_seconds_bucket[5m])) by (le)) > 10
```

Comment:
Do we need to do anything clever to avoid scraping deleted pods, @brancz?

Reply:
Nope that’s handled by Prometheus discovery :)